src/image_tools/bake.py

```diff
@@ -102,9 +103,21 @@ def bakefile_product_version_targets(
     tags = build_image_tags(image_name, args.image_version, versions["product"])
     build_args = build_image_args(versions, args.image_version)
     target_name = bakefile_target_name_for_product_version(product_name, versions["product"])
+    rfc3339_date_time = datetime.now(timezone.utc).isoformat()
+    revision = get_git_revision()
 
+    # The build-date label is set on UBI images automatically so we want to override it to not cause confusion even though it means we have the same date in the labels multiple times
     result = {
         target_name: {
+            "annotations": [
+                f"org.opencontainers.image.created={rfc3339_date_time}",
```

Comment:
Just FYI it might be the case that someone generates the bakefile with the script and after a period of time builds the actual image. But I think this is really really rare and therefore good enough

Reply:
it's probably also a cache buster